pandas/core/arrays/base.py

```diff
@@ -610,3 +620,14 @@ def _ndarray_values(self):
         used for interacting with our indexers.
         """
         return np.array(self)
+
+    # ------------------------------------------------------------------------
+    # Utilities for use by subclasses
+    # ------------------------------------------------------------------------
+    def is_sequence_of_dtype(self, seq):
```

Comment:
For what is this needed?

Reply:
Let's suppose that you don't implement the `ExtensionArray` operators as methods of the subclass of `ExtensionArray`, but you let the underlying `ExtensionDtype` handle the operators for you.  (This is what I used for `Decimal`).  Some of the operators will return a sequence containing all objects of `ExtensionDtype`.  Some operators (e.g., logical ones), will not.  So internally, it's useful to have a test to know whether a sequence has objects of the corresponding `ExtensionDtype` so that you can then return an `ExtensionArray` as a result, otherwise, you just let things get coerced based on the type in the sequence.